### PR TITLE
Minor styling tweaks

### PR DESCRIPTION
**Pull request by BenjaminVerble**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Like react for your data",
   "main": "src/index.js",
   "scripts": {
-    "test": "node test/test.js"
+    "test": "node test/test.js && standard"
```
Comment (dlmanning) [inline package.json]: I'm fine with using standard, but could we use eslint directly instead?

**Comment by dlmanning** [comment]
This all looks good except for the note about standard and eslint. Look at: https://www.npmjs.com/package/eslint-config-standard

File: package.json (hunk 1 above)
Comment (BenjaminVerble) [inline package.json]: that's fine. I'll swap it out.

**Comment by dlmanning** [comment]
Thanks!

**Pull request merged**